src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1015,8 +1016,29 @@ public Task ExecuteCommandAsync(PSCommand psCommand)
         {
             Validate.IsNotNull(nameof(scriptString), scriptString);
 
+            PSCommand command = null;
+            if(CurrentRunspace.Runspace.SessionStateProxy.LanguageMode != PSLanguageMode.FullLanguage)
+            {
+                try
+                {
+                    var scriptBlock = ScriptBlock.Create(scriptString);
+                    PowerShell powerShell = scriptBlock.GetPowerShell(isTrustedInput: false, null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename 'powerShell' which hides the field with the same name.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5481868)